Cargo.toml

```diff
@@ -6,34 +6,35 @@ edition = "2021"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-egui_winit_vulkano = "0.22.0"
-vulkano-shaders = "0.32.0"
-vulkano-win = "0.32.0"
-vulkano-util = "0.32.0"
-egui-winit = "0.20.1"
-bytemuck = "1.8.0"
-vulkano = "0.32.3"
+egui_winit_vulkano = "0.24.0"
+vulkano-shaders = "0.33.0"
+vulkano-win = "0.33.0"
+vulkano-util = "0.33.0"
+egui-winit = "0.21.1"
+bytemuck = "1.13.1"
+vulkano = "0.33.0"
 kdmapi = { git = "https://github.com/arduano/kdmapi.git", rev = "9329cfc" }
-egui = "0.20.1"
-winit = "0.27.3"
-rayon = "1.5.3"
+egui = "0.21.0"
+winit = "0.28.3"
+rayon = "1.7.0"
 midi-toolkit-rs = { git = "https://github.com/arduano/midi-toolkit-rs.git", rev = "cff22ac" }
 xsynth-core = { git = "https://github.com/arduano/xsynth.git", rev = "4b417da" }
 xsynth-realtime = { git = "https://github.com/arduano/xsynth.git", rev = "4b417da" }
 gen-iter = "0.3.0"
-enum_dispatch = "0.3.8"
-palette = "0.6.0"
-crossbeam-channel = "0.5.5"
+enum_dispatch = "0.3.11"
+palette = "0.7.1"
+crossbeam-channel = "0.5.8"
 rand = "0.8.5"
 confy = "0.5.1"
-serde_derive = "1.0.145"
-serde = "1.0.145"
-toml = "0.6.0"
+serde_derive = "1.0.160"
+serde = "1.0.160"
+toml = "0.7.3"
 colors-transform = "0.2.11"
-directories = "4.0.1"
+directories = "5.0.0"
 rustc-hash = "1.1.0"
 atomic_float = "0.1.0"
-egui_file = { git = "https://github.com/Kaydax/egui_file.git", rev = "7ac26fb" }
+egui_file = { git = "https://github.com/StratusFearMe21/egui_file.git", rev = "f8bd63b" }
+glidesort = "0.1.2"
```

Comment:
Do we really need this? It's used to sort a single 256 item array each frame, `sort_unstable_by_key` should be enough for that.